libs/shared-entity/src/error.rs

```diff
@@ -101,9 +106,27 @@ impl From<serde_json::Error> for AppError {
   }
 }
 
+impl From<opener::OpenError> for AppError {
```

Comment:
You can use #[cfg(feature = "cloud")] to isolate the dependencies from client_api crate

Reply:
I see.  I have made the changes